app/models/participant.rb

```diff
@@ -0,0 +1,9 @@
+class Participant < ApplicationRecord
+  include TranslateEnum
+
+  belongs_to :user, class_name: "User", foreign_key: :user_id, inverse_of: :participants
+  belongs_to :melange, inverse_of: :participants
+
+  enum role: {organizer: 0, moderator: 5, guest: 10}
```

Comment:
Wartości enum daj w String'ach

```suggestion
  enum role: {organizer: "organizer", moderator: "moderator", guest: "guest"}
```